src/crl.rs

```diff
@@ -52,6 +52,86 @@ pub struct CertRevocationList<'a> {
     pub crl_number: Option<&'a [u8]>,
 }
 
+/// Representation of a RFC 5280[^1] profile Certificate Revocation List (CRL) revoked certificate
+/// entry.
+///
+/// [^1]: <https://www.rfc-editor.org/rfc/rfc5280#section-5>
+pub struct RevokedCert<'a> {
+    /// Serial number of the revoked certificate.
+    pub serial_number: &'a [u8],
+
+    /// The date at which the CA processed the revocation.
+    pub revocation_date: Time,
+
+    /// Identifies the reason for the certificate revocation.
+    pub reason_code: Option<RevocationReason>,
```

Comment:
Given the comment for `Unspecified`, should we make this non-`Option`al instead?

Reply:
That sounds reasonable. Maybe add a comment to that effect?